dist-persist/wbstack/src/Internal/WbStackPlatformReservedUser.php

```diff
@@ -131,10 +131,30 @@ public static function getOAuthConsumer($consumerName, $version) {
             return false;
         }
 
-        return [
+        $data = [
             'agent' => $c->getName(),
             'consumerKey' => $c->getConsumerKey(),
             'consumerSecret' => \MediaWiki\Extension\OAuth\Backend\Utils::hmacDBSecret( $c->getSecretKey() ),
         ];
+
+        if ( $includeAccess ) {
```

Comment:
My hacky way of testing this locally would be to short circuit this if statement and always execute the branch, then see if Magnustools Oauth consumers stil work and also receive the additional data.

Reply:
And you'd be happiest if Magnustools does not get this additional data?

My understanding is that we wouldn't want Magnus tools to get these credentials since they would allow editing as the platform reserved user

Reply:
> And you'd be happiest if Magnustools does not get this additional data?

Hence the additional parameter, yes